google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -118,35 +108,43 @@ def create(
                 no impact on whether the port is actually exposed, any port listening on
                 the default "0.0.0.0" address inside a container will be accessible from
                 the network.
-
-        Returns:
-            local model: Instantiated representation of the local model.
         """
-        env = None
-        ports = None
-
-        if serving_container_environment_variables:
-            env = [
-                gca_env_var_compat.EnvVar(name=str(key), value=str(value))
-                for key, value in serving_container_environment_variables.items()
-            ]
-        if serving_container_ports:
-            ports = [
-                gca_model_compat.Port(container_port=port)
-                for port in serving_container_ports
-            ]
+        if serving_container_spec:
+            if not serving_container_spec.image_uri:
+                raise ValueError(
```

Comment:
Add a "Raises" docstring section for this error

Reply:
Done.